recipes/orekit/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+$PYTHON -m jcc \
+--use_full_names \
+--python orekit \
+--version 9.0 \
+--jar $SRC_DIR/orekit-conda-recipe/orekit-9.0.jar \
```

Comment:
Please replace `9.0` with `${PKG_VERSION}`. That way it will be easier to update later.